slash/css/src/Message/Message.css

```
@@ -0,0 +1,80 @@
+.af-alert {
```

vue que alert et déprécier il faudrait utiliser le nom message tout en gardant l'ancien pour limiter le risque de break.
```suggestion
.af-message,
.af-alert {
```

---

mettre la classe dans le css, ca résoud pas le break. Il faut que le composant porte la classe pour eviter le breaking change. Parce que si dans mon app je target `.af-alert { background: purple}` dans mon css, et que le componsant n'a plus la classe, ca fonctionne plus

---

j'avais pas pensez à ca bien vue, par contre il faudra prévoir de remplacer les classes pour être cohérent avec les autres composant.